app/views/layouts/_communicart_header.html.erb

```diff
@@ -25,7 +25,9 @@
     <%- if signed_in? %>
       <div class="mycart-link">
         <ul class="header-nav">
-            <li><%= link_to("My Requests", carts_path) %> <%= image_tag "img-cart.png", width: '18px', height:'16px' %></li>
+          <li><%= link_to "My Requests", carts_path %> <%= image_tag "img-cart.png", width: '18px', height:'16px' %></li>
+          <%# TODO only show to NCR people %>
+          <li><%= link_to "New Request", new_ncr_proposal_path %></li>
```

Comment:
That's a good question. It certainly seems like users need to be restricted to a domain/client/use case (or perhaps multiple).

When this was implemented at a previous employer, we had different schemas for each client. Basically, all requests included a header which provided the client id; that client id then determined which db connection was used. This acted as a form of context throughout the rest of the request. That might be overkill here, but it'd certainly be a good idea to separate which models are associated with a single client (carts, approvals), which could have multiple clients (perhaps users), and which are system wide (perhaps none, right now?)